src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -428,12 +439,11 @@ internal static bool IsEdgeSupportedType(Type type)
                 type = type.GetElementType();
             }
 
-            if (type == typeof(int) ||
-                type == typeof(double) ||
+            if (type.IsPrimitive ||
```

Comment:
Ah, I didn't realize Edge was doing a ToString on those types, so ultimately, what is provided to the script doesn't change.